notify: add markdown support for wechat (#2309)
prometheus/alertmanager: docs/configuration.md

# API request data as defined by the WeChat API.
[ message: <tmpl_string> | default = '{{ template "wechat.default.message" . }}' ]

> # The message type only support for `text` and `markdown`.

Reviewer: ```suggestion
# Type of the message type, supported values are `text` and `markdown`.
```
